README.md

```diff
@@ -1,2 +1,23 @@
-# etl-samples
-Sample ETL scenarios written in Ballerina
\ No newline at end of file
+# ETL Samples
+This repository contains various code samples for ETL tasks written in Ballerina.
+
+## Subdirectories
+Each subdirectory contains relevant code and instructions for the corresponding task. Explore the subdirectories for detailed code samples and documentation.
+
+#### Data Extraction (E)
+- [Extract Data from Database](extract-data-from-database): Sample code for extracting customer orders from database.
+- [Extract Information From Unstructured Data](extract-information-from-unstructured-data): Sample code for extracting required information from unstructured data using AI.
+- [Extract Data from GitHub](extract-data-from-github): Sample code samples for extracting data from GitHub.
+
+#### Data Transformation (T)
+- [Remove Duplicates](remove-duplicates): Sample code for deduplication.
+- [Remove Approximate Duplicates](remove-approximate-duplicates): Sample code for approximate deduplication.
+- [Error Correction](error-correction): Sample code for error correction in data.
+- [Clean and Standardize Data](clean-and-standardize-data): Sample code for cleaning and standardizind data.
```

Comment:
@LakshanWeerasinghe should be standardizing

Reply:
updated.